src/runtime-rs/crates/hypervisor/Cargo.toml

```diff
@@ -39,7 +41,7 @@ safe-path = "0.1.0"
 crossbeam-channel = "0.5.6"
 
 [features]
-default = []
+default = ["cloud-hypervisor"]
```

Comment:
Seems we should not open "cloud-hypervisor" feature by default.

Reply:
fixed